benchmarks/BenchmarkResultsRaw.md

```diff
@@ -0,0 +1,36 @@
+# Benchmark results
```

Comment:
nit: Missing trailing newline at end of file. This probably also deserves a summary or being in a single docs page with collapsible sections